timescale/init.go

```diff
@@ -38,6 +38,13 @@ func Migration() migrate.MemoryMigrationSource {
 				},
 				Down: []string{`SELECT remove_retention_policy('telemetry');`},
 			},
+			{
+				Id: "telemetry_3",
+				Up: []string{
+					`ALTER TABLE telemetry add mac_address TEXT;`,
```

Comment:
should not be allowed to be null

Reply:
I'm concerned in some cases we will not be able to access the MAC address (say, firmware may block it), so that's why I did not limit.